lib/galaxy/tools/verify/__init__.py

```diff
@@ -91,6 +91,10 @@ def verify(
         # if the server's env has GALAXY_TEST_SAVE, save the output file to that dir
         if keep_outputs_dir:
             ofn = os.path.join(keep_outputs_dir, filename)
+            try:
+                os.makedirs(os.path.dirname(ofn))
+            except OSError:
+                pass
```

Comment:
What about:
```python
            # filename can contain a subdirectory part
            ofd = os.path.dirname(ofn)
            if not os.path.isdir(ofd):
                os.makedirs(os.path.dirname(ofn))
```
so that `OSError`s different from "directory already exists" are not silenced.

Reply:
good idea